_episodes/l1-01-collaborating_with_git.md

```diff
@@ -25,26 +44,36 @@ Even when working independently you might find you need to need to work on
 different things at different times. This is greatly compounded however when you
 have multiple developers all wanting to contribute to the same Git repository.
 
-Today we will see how Git allows you to have multiple simultaneous streams of
-work via **branching and merging**. You can use branches to organise your
-individual work and as a way to ensure your Git history doesn't clash with other
-contributors.
+We will see how Git allows you to have multiple simultaneous streams of work via
+**branching and merging**. You can use branches to organise your individual work
+and as a way to ensure your Git history doesn't clash with other
+contributors. The ability to merge branches even supports working on the same
+part of the code as somebody else so you can work on whatever you want and worry
+about sorting out conflicts later.
 
-### Need to Coordinate Efforts
+### Different Points of View
 
-Developer 1 - "I'm still waiting on those changes to the data analysis workflow."
+Developer 1 - "Here's what I've been working on for the last month."
 
-Developer 2 - "Huh? I added those a month ago."
+Developer 2 - "Hmmm... if we tweak things here then it might be faster."
 
-TODO
+Two heads, as they say, are better than one and writing software is no
+exception. There is no greater benefit to collaboration than being able to pick
+someone else's brain about a problem. In software development this is usually
+called peer review and it's considered good practice for all code to be
+independently looked over.
 
-### Access and Permissions
+GitHub provides functionality for peer review via **Pull Requests**.
 
-Developer 1 - "You can put your changes here."
+### Need to Coordinate Efforts
 
-Developer 2 - "It says permission denied :-("
+Developer 1 - "I'm still waiting on those changes to the data analysis workflow."
 
-TODO
+Developer 2 - "Huh? I added those a month ago."
+
+Successfully coordinating the efforts of multiple contributors is a key
+challenge to avoid delay and duplication of work. GitHub can help here via
+**Issues** that track planned, on-going and completed work and who is doing it.
```

Comment:
Issues are mentioned in the introductory course (@TinyMarsh work). Will they be revisited here in the context of collaborating with others?